notifier/icons.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"sync"
+
+	"github.com/safing/portmaster-ui/notifier/icons"
+)
+
+var (
+	appIconEnsureOnce = new(sync.Once)
+	appIconPath       string
+)
+
+func ensureAppIcon() (location string, err error) {
+	appIconEnsureOnce.Do(func() {
+		if appIconPath == "" {
+			appIconPath = filepath.Join(dataDir, "exec", "portmaster.png")
+		}
+		err = ioutil.WriteFile(appIconPath, icons.PNG, 0o0644)
```

Comment:
you actually state that portmaster processes should never read from the `exec` dir ....

Reply:
Discussed: Due to lack of better (and quick) alternatives, we will stick with this for now.
Long-term the Portmaster could maintain icons files in the main directory.